src/ipmi.c

```diff
@@ -287,6 +325,13 @@ static int sensor_list_add(ipmi_sensor_t *sensor) {
   }
   } /* switch (sensor_type) */
 
+  /* if sensor provides the percentage value, add "_percent" suffix to the
+   * sensor collectd type */
+  if (ipmi_sensor_get_percentage(sensor)) {
+    ssnprintf(percent_type, sizeof(percent_type), "%s_percent", type);
```

Comment:
Please use `snprintf()`. The `ssnprintf()` wrapper has been removed from *master*.